students/chaycasso/Lesson04/file_lab.py

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python3
+#
+import pathlib
```

Comment:
i think it might be better to `import os` and use `os.path.` and the functions after that.  though to be honest haven't used pathlib